java-bootcamp-2018-tdd/src/main/java/com/globant/rest/UserController.java

```diff
@@ -0,0 +1,93 @@
+package com.globant.rest;
+
+import java.util.List;
+
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.globant.UserService.IUserService;
+import com.globant.UserService.UserProfile;
+import com.globant.UserService.UserService;
+import com.globant.UserService.UserServiceProxy;
+/**
+ * rest controller for web service
+ *
+ */
+@RestController

```

Comment:
Too much GET and no other methods, REST core principle is that HTTP methods represent the different possible operations.
You can do something like
@RequestMapping(value = "/ex/foos", method = POST)

or

use the annotations like

GetMapping, PutMapping, DeleteMapping, PatchMapping, RequestMapping